packages/safe-notification-service/src/services/confirm-transaction.interface.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Interface for confirming Safe transactions
+ */
+export interface IConfirmTransactionService {
+    confirmTransaction(input: {
+        hashToApprove: string,
+        signatureData: string,
+        safeAddress: string,
+        ethAddress: string,
+    }): Promise<void>
```

Comment:
Nitpick: wonder if we could return something else here for better error handling such as success boolean and\or tx hash